src/kms-message/src/hexlify.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018-present MongoDB, Inc.
```

Comment:
Files in `src/kms-message` are copied from libmongocrypt. Suggest reverting changes in `src/kms-message`. Reverting may fix the [kms-divergence-check](https://spruce.mongodb.com/task/mongo_c_driver_smoke_kms_divergence_check_patch_e87940820c7c643172e7d005484e7cf1ee824bec_667c31d57a87820007728314_24_06_26_15_20_55?execution=0) task.

Reply:
Done.